hr_expense_receipt_require/tests/test_hr_expense.py

```diff
@@ -3,44 +3,51 @@
 
 
 class TestHrExpense(common.TransactionCase):
-
     def setUp(self):
         super(TestHrExpense, self).setUp()
 
-        self.product_expense = self.env['product.product'].create({
-            'name': 'Flight Ticket',
-            'standard_price': 700,
-            'list_price': 700,
-            'type': 'service',
-            'default_code': 'CONSU-DELI-COST',
-            'taxes_id': False,
-            'expense_receipt_required': True
-        })
+        self.product_expense = self.env["product.product"].create(
+            {
+                "name": "Flight Ticket",
+                "standard_price": 700,
+                "list_price": 700,
+                "type": "service",
+                "default_code": "CONSU-DELI-COST",
+                "taxes_id": False,
+                "expense_receipt_required": True,
+            }
+        )
 
     def test_flight_expense_submission_without_attachment(self):
-        flight_expense = self.env['hr.expense'].create({
-            'name': 'Flight Expense',
-            'product_id': self.product_expense.id,
-            'unit_amount': 100.0,
-            'quantity': 1,
-            'employee_id': self.env.ref('hr.employee_hne').id
-        })
+        flight_expense = self.env["hr.expense"].create(
+            {
+                "name": "Flight Expense",
+                "product_id": self.product_expense.id,
+                "unit_amount": 100.0,
+                "quantity": 1,
+                "employee_id": self.env.ref("hr.employee_hne").id,
+            }
+        )
         with self.assertRaises(UserError):
             flight_expense.action_submit_expenses()
 
     def test_flight_expense_report_submission_without_attachment(self):
```

Comment:
Can you please add a test case covering the new use case added here?
You can copy and modify this one.